aladdin/lib/k8s/helm.py

```diff
@@ -86,16 +86,27 @@ def pull_packages(self, project_name, publish_rules, git_ref, extract_dir):
                     pass
 
     def find_values(self, chart_path, cluster_name, namespace):
+        """
+        Find all possible values yaml files for override in increasing priority
+        Values and overrides are defined/specified in helm args in a specific order
+        1. project values.yaml (picked up by helm automatically)
+        2. project cluster values.yaml
+        3. project cluster namespace values.yaml
+        4. site.yaml file (on local)
+        5. aladdin-config default `values.yaml`
+        6. aladdin-config cluster `values.yaml`
+        7. aladdin-config cluster namespace `values.yaml`
+        8. user passed overrides
```

Comment:
may be worth changing the order of priority here, but that is a larger discussion/effort